src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -123,6 +123,9 @@
   <data name="OutOfMemory" xml:space="preserve">
     <value>Out of process memory.</value>
   </data>
+  <data name="UnsupportedOSForRemoteEnumeration" xml:space="preserve">
+    <value>Remote PSSession enumeration with -ComputerName is only supported on Windows and not "{0}".</value>
```

Comment:
Makes no sense to point OS version. Please remove:
```suggestion
    <value>Remote PSSession enumeration with -ComputerName is only supported on Windows.</value>
```

Reply:
I find it provides a few extra debug details and doesn't hurt to include. It's also somewhat helpful to identify where the cmdlet was run when it comes to remoting ones as people may be confused if the OS platform is local or remote based. Including the local OS information in the error helps to avoid that ambiguity.